### PR TITLE
[fix] Centered Layout demo

### LINKED ISSUE #1283: Demos - no "Centered Layout"
image

### PR DESCRIPTION
**Pull request by ibelar**
fix: #1283

### CODE REVIEW

File: demos/layout/layouts.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 $buttons = [
     ['page' => ['layouts_nolayout'],               'title' => 'HTML without layout'],
     ['page' => ['layouts_manual'],                 'title' => 'Manual layout'],
-    ['page' => ['/demos/basic/header', 'layout' => \atk4\ui\Layout\Centered::class], 'title' => 'Centered layout'],
+    ['page' => ['/demos/basic/header', 'layout' => 'Centered'], 'title' => 'Centered layout'],
```
Comment (mvorisek) [inline demos/layout/layouts.php]: This does not work - I think it is some leftover from some not properly done merge... Should be probably solved like any other layout demos - like `layouts_manual` etc., ie. no `'layout' => ...` at all.

**Review by mvorisek — CHANGES_REQUESTED** [review]

File: demos/layout/layouts.php (hunk 1 above)
Comment (ibelar) [reply demos/layout/layouts.php]: Sure does. The layout is load via $_GET argument:
https://ui.agiletoolkit.org/demos/basic/header.php?layout=Centered

**Review by georgehristov — APPROVED** [review]

File: demos/layout/layouts.php (hunk 1 above)
Comment (mvorisek) [reply demos/layout/layouts.php]: The issue was absolute URL there.

Comment (mvorisek) [reply demos/layout/layouts.php]: image

@ibelar there are still two more absolute paths in the project to fix

Comment (ibelar) [reply demos/layout/layouts.php]: these are separate issues. Let merge this and I will check the others laters on.

Comment (mvorisek) [reply demos/layout/layouts.php]: I created an issue for that - https://github.com/atk4/ui/issues/1289 - and merging this one.

**Review by mvorisek — APPROVED** [review]

**Pull request merged**